input/pagecontent/GFI-004.md

```diff
@@ -0,0 +1,145 @@
+
+In order to proof possession of the credentials, the holder uses its private key to sign the Verifiable Presentation.
+
+This interaction is scoped on the interaction between two computers with not necessarily a human in the loop. Therefore the interaction is a back-channel interaction without a user-agent such as a web browser. For this reason the OAuth 2.0 Extension [RFC 7523](https://datatracker.ietf.org/doc/html/rfc7523) is used to offer the credentials in the form of a JWT assertion directly to the token endpoint without the use of a authorization request.
+
+### Actor Roles
+
+| Actor                           | Role                                                                                      |
+| ------------------------------- | ----------------------------------------------------------------------------------------- |
+| Holder (Client)                 | Requests an access token from the authorization server by offering verifiable credentials |
+| Verifier (Authorization Server) | Verifies the offered credentials and issues an access token                               |
+
+### Referenced standards
+
+- [OAuth 2.0 Authorization Framework](https://datatracker.ietf.org/doc/html/rfc6749)
+- [RFC 7523: JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants](https://datatracker.ietf.org/doc/html/rfc7523)
+- [Verifiable Credentials Data Model 1.1](https://www.w3.org/TR/2022/REC-vc-data-model-20220303/)
+- [OAuth 2.0 Demonstration of Proof-of-Possession (DPoP)](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop-04)
+
+### Messages
+
+1. Access Token Request
+2. Access Token Response
+
+<div>
+{% include GFI-004.svg %}
+</div>
+
+#### Access Token Request
+
+##### Trigger events
+
+- A client needs to obtain an access token from an authorization server.
+
+##### Message semantics
+
+The client requests an access token from the authorization server using the OAuth 2.0 JWT bearer token flow as defined in [RFC 7523](https://datatracker.ietf.org/doc/html/rfc7523).
+
+If an authorization server is shared between multiple (care) organizations (e.g. a multi-tenant setup), the authentication server must have a path parameter or a different endpoint to identify the organization the client is requesting the access token for. The client should not have to know the internal identifier of the organization. The access token must be bound to the requested organization.
+
+If the client wants to bind the access token to a private key, it can include a DPoP header in the request as defined in [OAuth 2.0 Demonstration of Proof-of-Possession (DPoP)](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop-04).
```

Comment:
```suggestion
If the client wants to bind the access token to a private key, it can include a DPoP header in the request as defined in [OAuth 2.0 Demonstration of Proof-of-Possession (DPoP)](https://datatracker.ietf.org/doc/html/rfc9449).
```